src/winston-logstash-latest.ts

```diff
@@ -14,7 +14,7 @@ import { IConnection, PlainConnection, SecureConnection } from "./connection";
 // Inherit from `winston-transport` so you can take advantage
 // of the base functionality and `.exceptions.handle()`.
 //
-module.exports = class LogstashTransport extends Transport {
```

Comment:
How does this affect backward compatibility? I have spent quite a lot of time keeping everything backwards compatible and it's important to understand if there are complications due to this change.

Reply:
It supposed to be no changes compared with the previous version, changing to a new export style due to the tsc compiler not recognizing the module.exports inside the file and does not export anything for types.